docs/howto/kubernetes-on-kind.md

```diff
@@ -0,0 +1,46 @@
+# Booting Kubernetes with Kubernetes in Docker (KinD)
+
+This tutorial will focus on using a tool called [kind](https://github.com/kubernetes-sigs/kind), also known as
+"Kubernetes IN Docker".
+
+If you haven't installed them already, go ahead and [install Docker](https://docs.docker.com/install/),
+[install kind](https://github.com/kubernetes-sigs/kind#installation-and-usage), and
+[install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+You'll need `kubectl` to interact with the cluster once it's created.
+
+## Create a cluster
+
+Once Docker, kind, and kubectl are installed, create a cluster with kind:
+
+```console
+$ kind create cluster
```

Comment:
Nit: Consider noting that this will create a single node cluster, which is perfect for the krustlet use case, but not necessarily for everything. Just a clarity thing